SerialSample/CPP/Generated Files/XamlTypeInfo.Impl.g.cpp

```diff
@@ -95,6 +95,41 @@ ::Windows::UI::Xaml::Markup::IXamlMember^ ::XamlTypeInfo::InfoProvider::XamlType
 }
 
 
+::Windows::UI::Xaml::Markup::IXamlType^ ::XamlTypeInfo::InfoProvider::XamlTypeInfoProvider::CheckOtherMetadataProvidersForName(::Platform::String^ typeName)
```

Comment:
the generated files shouldn't have been checked in to begin with ... please remove them if possible.